website/docs/r/monitor_action_group.html.markdown

```diff
@@ -122,7 +122,7 @@ The following arguments are supported:
 
 * `name` - (Required) The name of the Action Group. Changing this forces a new resource to be created. 
 * `resource_group_name` - (Required) The name of the resource group in which to create the Action Group instance. Changing this forces a new resource to be created.
-* `short_name` - (Required) The short name of the action group. This will be used in SMS messages.
+* `short_name` - (Required) The short name of the action group. This will be used in SMS messages. The length should be no less than 1 and no greater than 12.
```

Comment:
There's validation for this field, so we shouldn't need to explicitly document this here:

```suggestion
* `short_name` - (Required) The short name of the action group. This will be used in SMS messages.
```